django-backend/fecfiler/scha_transactions/migrations/0024_schatransaction_contact.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 3.2.12 on 2022-09-22 21:34
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+def add_new_contact_id(apps, schema_editor):
+    sch_a_transaction = apps.get_model("scha_transactions", "SchATransaction")  # noqa
+    contact = apps.get_model("contacts", "Contact")  # noqa
+
+    for trans in sch_a_transaction.objects.all():
+        contact_dict = {
+            'type': trans.entity_type,
+            'committee_account_id': trans.committee_account_id,
+            'street_1': trans.contributor_street_1,
+            'street_2': trans.contributor_street_2,
+            'city': trans.contributor_city,
+            'state': trans.contributor_state,
+            'zip': trans.contributor_zip
+        }
+        if trans.entity_type == 'IND':
+            contact_dict['last_name'] = trans.contributor_last_name
+            contact_dict['first_name'] = trans.contributor_first_name
+            contact_dict['middle_name'] = trans.contributor_middle_name
+            contact_dict['prefix'] = trans.contributor_prefix
+            contact_dict['suffix'] = trans.contributor_suffix
+            contact_dict['employer'] = trans.contributor_employer
+            contact_dict['occupation'] = trans.contributor_occupation
+        elif trans.entity_type == 'COM':
+            contact_dict['committee_id'] = trans.donor_committee_fec_id
+            contact_dict['name'] = trans.contributor_organization_name
+        elif trans.entity_type == 'ORG':
+            contact_dict['name'] = trans.contributor_organization_name
+        retval = contact.objects.create(**contact_dict)
+        trans.contact_id = retval.id
+        trans.save()
+
+
+def remove_contact_id(apps, schema_editor):
+    sch_a_transaction = apps.get_model("scha_transactions", "SchATransaction")  # noqa
+    for trans in sch_a_transaction.objects.all():
+        trans.contact_id = None
+        trans.save()
```

Comment:
I'm not following this part. Why are the contact_id references being removed from the transaction records?